ouroboros-consensus/bench/PerasCertDB-bench/Main.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE LambdaCase #-}
+
+-- | This module contains benchmarks for Peras chain weight calculation as implemented by
+--   the by the 'Ouroboros.Consensus.Storage.PerasCertDB.API.boostedWeightForFragment'
+--   function.
+--
+--   We benchmark the calculation on a static sequence of chain fragments of increasing
+--   length, ranging from 0 to around 2160, with a sampling rate of 100. The chain fragments
```

Comment:
In the ChainSync client, while syncing, we usually have fragments of length `k + 3*k = 8640` (because usually contain the entire volatile candidate (`k` headers) as well as one forecast worth of blocks (on average `3*k` headers).

Reply:
Good point, fixed!